docs/src/algebraic.md

```diff
@@ -260,7 +248,7 @@ is_less_root_order(a::QQBarFieldElem, b::QQBarFieldElem)
 
 **Examples**
 
-```jldoctest
+```jldoctest; setup = :(QQBar = algebraic_closure(QQ))
```

Comment:
I guess we could just search&replace this to:
````suggestion
```jldoctest
julia> QQBar = algebraic_closure(QQ);

````